packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -117,8 +117,9 @@ function Cell(props) {
       event.stopPropagation();
       event.preventDefault();
     } else if (onCellSelect) {
-      onCellSelect({ rowId, columnId }, { row: rowIndex, col: columnIndex });
+      onCellSelect({ rowId, columnId }, { row: rowIndex, col: columnIndex }, !!event.shiftKey, false);
       event.stopPropagation();
+      event.preventDefault();
```

Comment:
Why do we need to prevent the default behavior on mouse down?  It should not cause issues, right?

Reply:
It was added to prevent the default behavior of text selection. After our discussion offline, removed preventDefault() and used user-select to avoid shift click text selection.